app/src/renderer/components/staking/PageValidator.vue

```diff
@@ -205,21 +205,21 @@ export default {
       if (this.validator.revoked)
         return `This validator has been jailed and is not currently validating`
 
-      // status: candidate
+      // status: inactive
       if (parseFloat(this.validator.voting_power) === 0)
-        return `This validator has declared candidacy but does not have enough voting power yet`
+        return `This validator does not have enough voting power yet and is inactive`
```

Comment:
I think we need to clarify that inactive means that the validator is not on the active validator set:

`This validator does not have enough voting power yet and not on the active validator set`

Reply:
can an inactive validator be in the validator set?

Reply:
with inactive you also mean jailed ?

Reply:
i think we want to avoid the concept of the "validator set" - what people need to know is if a validator is validating or not (active vs. inactive) or if a validator has been jailed.

Reply:
other options:

1. `This validator is inactive because it does not have enough voting power to pass the validator set threshold`
2. `This validator  is inactive because it does not have enough voting power yet to participate on the consensus`

Reply:
they are good suggestions... but i don't know if we need to add extra concepts.